Sources/SwifterSwift/Foundation/MeasurementExtensions.swift

```diff
@@ -0,0 +1,52 @@
+// MeasurementExtensions.swift - Copyright 2021 SwifterSwift
+
+#if canImport(Foundation)
+import Foundation
+
+// MARK: - Methods
+@available(OSX 10.12, tvOS 10.0, watchOS 3.0, *)
+public extension Measurement where UnitType == UnitAngle {
+    /// SwifterSwift:  Create a `Measurement` for an angle with a specified value in degrees.
+    /// - Parameter value: The quantity of the angle in degree.
+    /// - Returns: Measurement for an angle with unit degrees.
+    static func degrees(_ value: Double) -> Measurement {
+        Measurement(value: value, unit: .degrees)
```

Comment:
I think we discussed in earlier PRs that functions won't elude `return` statements, only closures. WDYT @LucianoPAlmeida?

Reply:
Yeah, nice catch ... I think we should use explicit return here because that would be consistent with the rest of the project :)

Reply:
Modified.

BTW, is there any coding style / convention guideline for SwifterSwift?
I tried to dig repo history myself but there're always something missing. 🤔

Reply:
Thanks!

> BTW, is there any coding style / convention guideline for SwifterSwift?

I think we don't have a style, but normally the recommendation is just try to follow the same style as the rest of the codebase =]